templates/test/ci/cluster-template-prow-ci-version.yaml

```diff
@@ -166,13 +173,6 @@ spec:
       owner: root:root
       path: /tmp/kubeadm-bootstrap.sh
       permissions: "0744"
-    - contentFrom:
```

Comment:
was this change intentonal?

Reply:
nvm, it just moved above

Reply:
it's because of the strategic merge vs json patching strategies